xs/src/libslic3r/Print.cpp

```diff
@@ -230,7 +230,7 @@ Print::make_skirt()
         skirts = 1;
     
     const std::set<size_t> extruders{ this->extruders() };
-    auto extruder_it { extruders.begin() };
+    auto extruder_it = extruders.begin();
```

Comment:
Can this be cbegin() instead of begin()?

Reply:
Sure. AFAIK, `begin()` called on a `const` variable behaves as `cbegin()` anyway.

Reply:
Works for me.